pkg/pillar/cmd/ledmanager/ledmanager.go

```diff
@@ -670,6 +670,29 @@ func InitLedCmd(ledName string) string {
 	return ""
 }
 
+// MaxLEDBrightness
```

Comment:
revive expects a space (and words?) after the function name. But do actually don't need to export this function.

Reply:
Indeed. I've updated the PR.